### PR TITLE
#21 Create Metal Quartz Checkbox Component

### PR DESCRIPTION
**Pull request by pat270**

### CODE REVIEW

File: packages/metal-quartz-checkbox/src/MetalQuartzCheckbox.js [hunk 1]
```diff
@@ -19,6 +19,53 @@ class MetalQuartzCheckbox extends Component {
  * @type {!Object}
  */
 MetalQuartzCheckbox.STATE = {
+	checkboxInline: {
```
Comment (brunobasto) [inline packages/metal-quartz-checkbox/src/MetalQuartzCheckbox.js]: Hey @pat270,

From the name, it's not clear to me what this attribute does. So this is a list of objects with props for each checkbox?

Comment (pat270) [reply packages/metal-quartz-checkbox/src/MetalQuartzCheckbox.js]: Yes that is what it is, should look like when initializing from js:
```
checkboxInline: [{
	checked: true,
	label: 'Inline 1'
}, {
	label: 'Inline 2'
}, {
	checked: true,
	disabled: true,
	label: 'Inline 3'
}, {
	label: 'Inline 4'
}, {
	label: 'Inline 5'
}, {
	label: 'Inline 6'
}]
```
I was following what was being done at https://github.com/liferay-lexicon/lexicon-dropdown/blob/master/demos/index.html

**Comment by robframpton** [comment]
Hey @pat270 , what do you think about creating a base component called `MetalQuartzInputBase` that we can use to house the common attributes that most input types will use? 

In particular I'm thinking `name`, `value`, `disabled`, `label`, etc. That way we don't have to redefine those attrs for every input component we create.

**Comment by robframpton** [comment]
Due to f8abd7353e707302accfe8f6e644f3705ae98d61 this pr needs to be rebased.

**Comment by fernandosouza** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jbalsas** [comment]
I'm closing this for now. @carloslancha can pick it up as starting point whenever we get back to this again!

**Pull request closed without merging**